src/app/clusters/level-control/level-control.cpp

```diff
@@ -123,6 +123,145 @@ static bool shouldExecuteIfOff(EndpointId endpoint, CommandId commandId,
                                chip::Optional<chip::BitMask<LevelControlOptions>> optionsMask,
                                chip::Optional<chip::BitMask<LevelControlOptions>> optionsOverride);
 
+#ifdef EMBER_AF_PLUGIN_SCENES
+class DefaultLevelControlSceneHandler : public scenes::DefaultSceneHandlerImpl
+{
+public:
+    enum LevelControlEFS : uint8_t
```

Comment:
where is this used (I was about to ask why we need explicit uint8_t dependency, but trying to find usages I could find none.

Reply:
This is an artefact from when attributes used to be written in arrays of bytes, no longer necessary.